docs/griptape-framework/structures/config.md

```diff
@@ -112,7 +129,11 @@ serialized_config = custom_config.to_json()
 deserialized_config = AmazonBedrockStructureConfig.from_json(serialized_config)
 
 agent = Agent(
-    config=deserialized_config,
+    config=deserialized_config.merge_config({
+        "prompt_driver" : {
+            "model": "anthropic.claude-3-sonnet-20240229-v1:0",
+        },
+    }),
```

Comment:
Would be a good place to show off how `azure_deployment` can be merged in.

Reply:
updated on the azure openai example